projects/go-lib/src/lib/components/go-file-upload/go-file-upload.component.scss

```diff
@@ -0,0 +1,62 @@
+@import '../../../styles/variables';
+
+@mixin upload-active {
+  cursor: pointer;
+  background-color: $theme-light-bg-hover;
+  opacity: 0.8;
+}
+
+input[type="file"] {
+  visibility: hidden;
+}
+
+.go-file-upload {
+  height: 10rem;
+  border: 2px dashed grey;
```

Comment:
Can we change this to `1px dashed $theme-light-border`? Fits a bit better with the rest of the styles.